test/blackbox-tests/test-cases/github5555.t

```diff
@@ -0,0 +1,31 @@
+Using `binaries` in the default context of `env` is  not supported:
+
+  $ cat > dune-workspace << EOF
+  > (lang dune 3.1)
+  > (env (_ (binaries x.exe)))
+  > EOF
+
+  $ dune build
+  File "dune-workspace", line 2, characters 0-26:
+  2 | (env (_ (binaries x.exe)))
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Error: 'env' is only available since version 3.2 of the dune language. Please
```

Comment:
The error message is incorrect here. It error should mention `(env (_ (binaries ..)))` instead.